astropy/modeling/fitting.py

```diff
@@ -1128,20 +1128,21 @@ def objective_function(self, fps, *args):
         args : list
             [model, [weights], [input coordinates], {context}]
         context : dict
-            A dict of pre-computed parameters from the __call__method
+            A dict of pre-computed parameters from the ``__call__`` method.  All
```

Comment:
though `context` appears as a `dict` internally, documenting what amounts to arbitrary keyword arguments as a `dict` may be confusing to end users.

Reply:
As mentioned in https://github.com/astropy/astropy/pull/16982#discussion_r1751709514 maybe we can just list the kwargs directly